### PR TITLE
Falling blocks and better commands added

### PR DESCRIPTION
**Pull request by kweber1**

### CODE REVIEW

File: carpetmodSrc/carpet/commands/CommandLazyChunkBehavior.java [hunk 1]
```diff
@@ -25,38 +27,101 @@
...
 
     public String getName()
     {
         return "lazychunkbehavior";
     }
 
+    private void list(ICommandSender sender, String[] args) throws CommandException{
+        if (args.length != 1)
+        {
+            throw new WrongUsageException(getUsage(sender), new Object[0]);
```
Comment (Earthcomputer) [inline carpetmodSrc/carpet/commands/CommandLazyChunkBehavior.java]: Varargs means you don't need this parameter
```suggestion
            throw new WrongUsageException(getUsage(sender));
```

File: carpetmodSrc/carpet/commands/CommandLazyChunkBehavior.java [hunk 2]
```diff
@@ -25,38 +27,101 @@
...
+        }
+        LazyChunkBehaviorHelper.listLazyChunks(sender);
+    }
+    private void removeAll(ICommandSender sender, String[] args) throws CommandException{
+        if (args.length != 1)
+        {
+            throw new WrongUsageException(getUsage(sender), new Object[0]);
```
Comment (Earthcomputer) [inline carpetmodSrc/carpet/commands/CommandLazyChunkBehavior.java]: Same here

File: carpetmodSrc/carpet/commands/CommandLazyChunkBehavior.java [hunk 3]
```diff
@@ -25,38 +27,101 @@
...
     /**
      * Callback for when the command is executed
      */
     public void execute(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException
     {
         if (!command_enabled("CommandLazyChunkBehavior", sender)) return;
 
-        if (args.length != 2)
-        {
+        if(args.length<1 || args.length>3){
```
Comment (Earthcomputer) [inline carpetmodSrc/carpet/commands/CommandLazyChunkBehavior.java]: Can you add some spaces before and after the inequality signs?

**Review by Earthcomputer — CHANGES_REQUESTED** [review]

File: patches/net/minecraft/block/BlockFalling.java.patch [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+-                {
+-                    EntityFallingBlock entityfallingblock = new EntityFallingBlock(p_176503_1_, (double)p_176503_2_.func_177958_n() + 0.5D, (double)p_176503_2_.func_177956_o(), (double)p_176503_2_.func_177952_p() + 0.5D, p_176503_1_.func_180495_p(p_176503_2_));
+-                    this.func_149829_a(entityfallingblock);
+-                    p_176503_1_.func_72838_d(entityfallingblock);
+-                }
++                    if (!p_176503_1_.field_72995_K) {
++                        EntityFallingBlock entityfallingblock = new EntityFallingBlock(p_176503_1_, (double) p_176503_2_.func_177958_n() + 0.5D, (double) p_176503_2_.func_177956_o(), (double) p_176503_2_.func_177952_p() + 0.5D, p_176503_1_.func_180495_p(p_176503_2_));
++                        this.func_149829_a(entityfallingblock);
++                        p_176503_1_.func_72838_d(entityfallingblock);
++                    }
```
Comment (kerbaras) [inline patches/net/minecraft/block/BlockFalling.java.patch]: i don't understand this change in the patch, did identation changed? lets try keep the patch modifications as minimal as possible

**Review by kerbaras — CHANGES_REQUESTED** [review]

**Review by kerbaras — APPROVED** [review]
Fine for me, lets wait for earth to approve his review

**Review by Earthcomputer — APPROVED** [review]

**Pull request merged**